zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -4661,6 +4716,12 @@ components:
         - ACTIVE
         - COMPLETED
         - TERMINATED
+    FlowNodeInstanceStateEnum:
+      description: The state, one of ACTIVE, COMPLETED, TERMINATED.
```

🔧 Maybe description could say that these are flow node states. I guess `   ProcessInstanceStateEnum` could also use a better description :)

```suggestion
      description: Flow node states, one of ACTIVE, COMPLETED, TERMINATED.
```